app/models/__init__.py

```diff
@@ -886,6 +887,14 @@
     ]
 }
 
+
```

Comment:
Why was this block moved before `TEST_CASE_VALID_KEYS`?  Looks like this was done by mistake.

Reply:
Corrected